crates/voicevox_core_c_api/src/lib.rs

```diff
@@ -563,29 +732,36 @@ pub unsafe extern "C" fn voicevox_synthesizer_replace_mora_pitch(
     })())
 }
 
-/// `voicevox_synthesizer_synthesis` のオプション
+/// ::voicevox_synthesizer_synthesis のオプション。
 #[repr(C)]
 pub struct VoicevoxSynthesisOptions {
     /// 疑問文の調整を有効にする
     enable_interrogative_upspeak: bool,
 }
 
-/// デフォルトの `voicevox_synthesizer_synthesis` のオプション
+/// ::VoicevoxSynthesisOptions のデフォルト値。
 #[no_mangle]
 pub static voicevox_default_synthesis_options: VoicevoxSynthesisOptions = ConstDefault::DEFAULT;
 
-/// AudioQuery から音声合成する
-/// @param [in] synthesizer 音声シンセサイザ #VoicevoxSynthesizer
-/// @param [in] audio_query_json jsonフォーマットされた AudioQuery
-/// @param [in] style_id スタイルID #VoicevoxStyleId
-/// @param [in] options AudioQueryから音声合成オプション
-/// @param [out] output_wav_length 出力する wav データのサイズ
-/// @param [out] output_wav wav データの出力先
-/// @return 結果コード #VoicevoxResultCode
-///
-/// # Safety
-/// @param output_wav_length 出力先の領域が確保された状態でpointerに渡されていること
-/// @param output_wav 自動で output_wav_length 分のデータが割り当てられるので ::voicevox_wav_free で解放する必要がある
+/// AudioQueryから音声合成を行う。
+///
+/// 生成したwavを解放するには ::voicevox_wav_free を使う。
```

Comment:
<https://github.com/VOICEVOX/voicevox_core/pull/532/files#r1257278303>に続いてこっちもこうします。

```suggestion
/// 生成したWAVデータを解放するには ::voicevox_wav_free を使う。
```